lua/telescope/pickers.lua

```diff
@@ -1046,6 +1066,13 @@ function Picker:set_selection(row)
   end
 
   local entry = self.manager:get_entry(self:get_index(row))
+
+  local prompt_location = state.get_global_key "prompt_location"
+  if entry and prompt_location then
+    entry.lnum = prompt_location.row or 0
+    entry.col = prompt_location.col or 0
+  end
```

Comment:
not sure we should inject that here into the entry, i'd rather we just use that prompt_location in the actions, similar to what you did with the previewer, because with this right now you could have just used the `entry.lnum` and `entry.col` in previewer code, fyi

Why do you do it like this?

Reply:
To reuse the code of putting the cursor to the specific location, thus we will need to duplicate them but I’m find with that either